hub/util/exceptions.py

```diff
@@ -418,6 +418,12 @@ def __init__(self, expected: str, actual: str, htype: str):
         super().__init__(msg)
 
 
+class TensorMetaMutuallyExclusiveKeysError(MetaError):
+    def __init__(self, keys: List[str]):
+        msg = f"Following fields are mutually exclusive: {keys}"
+        super().__init__(msg)
```

Comment:
can you add a message parameter here? if im a user and specify both sample + chunk compression this message will be very confusing. 

i would just say something like "Specifying both sample-wise + chunk-wise compressions for a single tensor is not yet supported."